### ISSUE TITLE
tooltips not showing on initial load with start-opened and fixed-tooltip both set to true

### ISSUE THREAD

**Issue by legreco** [PygmySlowLoris/vue-fab#30]
My code:
```
  <fab
                main-tooltip="Actions"
...
                :start-opened="true"
                :position="position"    
                :actions="fabActions"
                fixed-tooltip=true
...
            
        ></fab>
```
Expected behavior:
1. On first load the fab is opened 
2. And  it shows the tooltip beside the actions 

Current behavior
1. On first load the fab is opened 
2. the tooltip is not showing up beside the actions 

Note
I have to close the fab and reopen it to have the tooltip showing up beside the actions.

Sorry for my bad english. And thanks for the help!

**Comment by Edujugon** [comment]
Hi @legreco ,

I'll release a new package version covering this functionality probably tomorrow.

I'll drop you a message here once it's done :)

**Comment by Edujugon** [comment]
Hey @legreco ,

We've just released a new package version (2.3.1) which should fix this issue.

Hope that helps :)

**Comment by Edujugon** [comment]
Hey @legreco ,
I'm closing this issue since it should be fixed with the latest release.

Feel free to reopent it if needed.
